frontend/src/metabase/dashboard/selectors.ts

```diff
@@ -383,6 +384,22 @@ export const getDashcardParameterMappingOptions = createCachedSelector(
   return props.card.id ?? props.dashcard.id;
 });
 
+// Embeddings might be published without passing embedding_params to the server,
+// in which case it's an empty object. We should treat such situations with
+// caution, assuming that an absent parameter is "disabled".
+export function getEmbeddedParameterVisibility(
```

Comment:
I'm not totally sure if we should have 2 selectors (for dashboard and question). Or have only 1 function that receives either an embedding resource or embedding parameters.

I think having only one function might be better because engineers could forget about another function when modifying one function. Both seem pretty coupled to me.

What do you think?

Reply:
You're right, it would be nice to avoid it. Unfortunately, we already have lots of duplication here and elsewhere for parameters and other parts of Query Builder and Dashboards.

So since this change is consistent with the rest, I'll keep it as is. The refactoring is coming but is outside the scope of this PR.